DI-1801/utils.py

```
@@ -0,0 +1,113 @@
+    -------
+    contents_list : list
+        list where each line in the file is an entry
+    """
+    with open(filename, mode="r", encoding="utf8") as f:
+        file_contents = f.read().splitlines()
+
+    # Remove any empty lines, leading and trailing whitespace and duplicates
+    contents_list = list(
+        set(
+            [item for item in (line.strip() for line in file_contents) if item]
+        )
```

Pylint (3.3.7)

[refactor] Consider using a set comprehension

(R1718)